### ISSUE TITLE
⌘1 Does not seem to work

### ISSUE THREAD

**Issue by iSapozhnik** [Clipy/Magnet#56]
Thanks for the great Framework! It's amazing!
However I ran into a problem that I could not solve by myself.
I'm trying to make a hotkey ⌘1 but it just does not work for me. I created a new test project, added Magnet package and in `viewDidLoad` implemented following:

```
override func viewDidLoad() {
        super.viewDidLoad()

        // Does not work
        if let keyCombo = KeyCombo(key: .one, cocoaModifiers: [.command]) {
            let cmdOne = HotKey(identifier: "CommandOne", keyCombo: keyCombo) { hotKey in
                print("CommandOne")
            }
            cmdOne.register()
        }

        // Works perfectly well
        if let keyCombo = KeyCombo(key: .a, cocoaModifiers: [.command]) {
            let cmdOne = HotKey(identifier: "CommandA", keyCombo: keyCombo) { hotKey in
                print("CommandA")
            }
            cmdOne.register()
        }
    }
```

If this makes any difference I'm testing this on Macbook Pro 15" Mojave 10.4.4 

Can you please support with that?

**Comment by Econa77** [comment]
@iSapozhnik Thanks for using the framework :)
I was aware of this problem myself and was in the process of fixing it. The mapping to keypad 1 is failing and will be fixed in this PR. 
https://github.com/Clipy/Sauce/pull/22

I only have a Japanese keyboard at the moment, so I bought an English keyboard on Amazon and am waiting for it to arrive. We will release it when we have confirmed that it works with English keyboard. Hold off on the update for a bit.

**Comment by iSapozhnik** [comment]
Great! Thanks for the update. I thought maybe it's due to my NSWindow configuration (I'm using custom implementation) :)

**Comment by Econa77** [comment]
@iSapozhnik We fixed it and released [v3.2.0](https://github.com/Clipy/Magnet/releases/tag/v3.2.0). If the problem arises again, please create an issue 👍

**Comment by iSapozhnik** [comment]
@Econa77 Works as expected! Thanks for the quick fix!
